exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Column.kt

```diff
@@ -39,6 +39,8 @@ class Column<T>(
 
     fun defaultValueInDb() = dbDefaultValue
 
+    internal var isGeneratedInDb: Boolean = false
```

Comment:
Could we check what names are used in other frameworks for that?

Reply:
Here is what I found:
- [Entity Framework - [DatabaseGenerated(DatabaseGeneratedOption.None)]](https://www.learnentityframeworkcore.com/configuration/data-annotation-attributes/databasegenerated-attribute)
- [Hibernate - @Generated](https://docs.jboss.org/hibernate/orm/6.2/javadocs/org/hibernate/annotations/Generated.html)

I didn't find this feature in Ktorm and jOOQ.

Reply:
`databaseGenerated` lgtm